ultravox/data/datasets.py

```diff
@@ -1142,10 +1143,12 @@ def __init__(
         self._static = static
 
         self._stop_strategy = stop_strategy
+        relative_frequencies = [
```

Comment:
I think life might be easier if we didn't try to read the `multiplier` value out of each dataset class, and instead read it from a config passed to the interleave class, similar to how this is done with `probabilities` in HF's `interleave_datasets`: https://huggingface.co/docs/datasets/en/package_reference/main_classes#datasets.interleave_datasets

Reply:
Then we wouldn't have to worry about the interaction with `len`, etc

Reply:
Then we'd need to calculate the probabilities by hand in the config? I think we'd need len either way because that's how hf determines the # of steps to take in epoch mode.

Reply:
Well, the weights are being set by hand already, right? It just seems a bit strange to have to read the weight/multiplier property from the dataset class, especially when the dataset class doesn't use it internally.